sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_PN="spatialindex-src"
+MY_P="${MY_PN}-${PV}"
```

Comment:
`MY_PN` is nowhere used except here, so you might as well just set `MY_P="spatialindex-src-${PV}"`

Reply:
DONE